src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -84,6 +84,7 @@ $overlayShapeZIndex: 1;
   font-size: $font-size-l;
   line-height: 1.188;
   margin: 14px 0 0 0;
+  word-wrap: anywhere;
```

Comment:
Can you switch it to the standard version, which is [`overflow-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap)? The `anywhere` value landed in Firefox 65 which is pretty recent but that's probably fine.